ThreatDragonModels/ImagingApp/ImagingApp.json

```diff
@@ -0,0 +1,50 @@
+{
+  "summary": {
+    "title": "ImagingApp",
```

Comment:
❔ Curious, did you think about going away from the "App" terminology. Maybe it still makes sense from a threat model perspective but figured if we are starting new it would be a good time to consider a change on it. I don't have anything better at the moment :-)

Reply:
I'm not against doing that, if we have no objections from other reviewers.